pages/dkp/kommander/2.1/release-notes/2.1.0/index.md

```diff
@@ -202,6 +202,22 @@ flux reconcile -n kommander-flux source git management --kubeconfig MANAGED_KUBE
 ✔ fetched revision main/GIT_HASH
 ```
 
+### Cert-manager expiration workaround
+
+`cert-manager` renews all certificates 60 days after you install Kommander on your cluster. Unfortunately, some applications or pods fail to receive the renewed certificate information, causing them to break upon expiration (90 days after Kommander was installed, which normally coincides with the date you created the cluster). As a result, your cluster stops running, and sometimes, you are unable to access the UI.
+
+D2iQ provides a workaround that forces the applications to reconcile and recognize the renewed certificate. This workaround extends the validity of the certificates to 10 years, fixes the certification reload issue, and restarts the affected pods once the new certificate is issued.
+
+It applies to any environment setup (networked, air-gapped, on-prem, etc.) and fixes the issue regardless of your issuer type ([SelfSigned][selfsigned] for air-gapped environments, [ACME][acme], or your own certificate issuer configured separately for your institution).
+
+To **prevent your applications from breaking**, or to **get the nodes up and running again**, and fix this issue permanently, run this command:
+
+<p class="message--important"><strong>IMPORTANT: </strong>If you have changed the default location of your <code>kubeconfig</> file, replace <code>~/.kube/config</code> with the absolute path of your file's location. For example, use <code>/home/example/my-kubeconfig.yaml<code> instead of <code>my-kubeconfig.yaml</code>.</p>
+
+```bash
+docker run -v ~/.kube/config:/kubeconfig -e KUBECONFIG=/kubeconfig mesosphere/rotate-certificate-hotfix:2.1.1
```

Comment:
I'm a little worried a customer might not read this IMPORTANT note closely and try to just copy-paste the command first, which has the potential for causing a lot of confusion or messing up the wrong cluster.

Also, none of our documentation or even the `dkp cli` currently merge the kubeconfig context into ~/.kube/config.   So I am just not sure having this command 'default' to using ~/.kube/config is a good choice.

I'd like to suggest we change this to something that will not work if you just blindly copy-pasta it.  For example:

```suggestion
<p class="message--important"><strong>IMPORTANT: </strong>Before running this command, you must replace <code><path/to/my_kubeconfig></code> with an absolute path of the location that contains the kubeconfig for the cluster you wish to update.   It will not work properly with a relative file path.  For example, use <code>/home/example/my-kubeconfig.yaml<code>, or <code>`pwd`/my-kubeconfig.yaml</p>

```bash
docker run -v <path/to/my_kubeconfig>:/kubeconfig -e KUBECONFIG=/kubeconfig mesosphere/rotate-certificate-hotfix:2.1.1
```

If you try to just copy-pasta this command, you get:

$ docker run -v <path/to/my_kubeconfig>:/kubeconfig -e KUBECONFIG=/kubeconfig mesosphere/rotate-certificate-hotfix:2.1.1
-bash: path/to/my_kubeconfig: No such file or directory

Which seems perfect to me - it tells you exactly what is broken...

Reply:
We could also change the ```bash to ```sh to avoid copy-paste. But I would hate to write the whole thing manually if I were a customer. Any thoughts?

Reply:
I think we decided we want the customer to be able to copy-paste as the command is complicated, but we want them to have to 'fix it' for their situation, so 'bash' is perfect.